azure/errors_test.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2023 The Kubernetes Authors.
```

Reply:
As best as I can tell, this file was originally added in 2019. Not sure exactly what it means that there wasn't an explicit copyright on it before then, but I think it's fine as-is.

Reply:
Gotcha, I just wasn't sure if the copyright header is allowed to be retroactive or not